### DISCUSSION TITLE
Liability and already loaded in accounts

### DISCUSSION THREAD

**Discussion by shadowjustice** [firefly-iii/firefly-iii#9563] [Firefly III] [answered]
Story time:
I have loaded up all my transactions from my bank account as a first time user of FF3. 
Now  I am defining all my bills and I would like to add my mortgage with the liability thing. 
I get:
![image](https://github.com/user-attachments/assets/8030bab1-9bba-4e2a-a3c3-d75ae6c620f3)

Sadly I can not do I assume because the IBAN is already linked to an expense account. 

Is my assumption correct? If  you import statements you probably would want control to what kind of account it is. 

How would I go about on fixing this? 

Updating the database is a possibility since I am an IT guy and I have a view on the database record which shows it is an expense account.
I am afraid though that just updating the type would mess up stuff.

EDIT:
Also for some reason sometimes it does not give this error and when I want to submit the liability I go to the expense account. To be fair this is mostly only a 1 time thing afterwards just in-use error.

**Comment by JC5** [comment, marked as answer]
You can edit the expense account and clear out the IBAN. I would then create a new liability with the IBAN.

**Reply by shadowjustice** [reply]
This is indeed the answer I have found a bug though:
1. Liability created under name "Elan SA House"
2. I can assign 1 transaction from "Elan SA" (original one) to the liability
3. When I try to bulk assign I can not change "To" From Elan SA  > Elan SA House (at least it does not auto fill)

Single edit behaviour <> bulk edit for the to field

Okay I tried. Just copying the name even it did not auto fill it but that also works. No bug just a little change in behaviour.
